07.bowling_object/bowling_object.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+def main
+  score = ARGV[0]
+  scores = score.split(',')
+  shots = []
```

Comment:
`Game.new(shots)`に入れて`initialize(frames)`で受け取っているので、これは`framesという変数が正しいですかね？
```suggestion
  frames = []
```

Reply:
ただ、フレームは1ゲームに10個なのでデータの持ち方としては以下のような感じになるかな
```ruby
[
  [1,2],
  [X],
  [4,6],
  [7,1],
  ...
]
```

構造的には以下みたいになると綺麗

```
Game [
  Frame [ Shot ]
]
```

Reply:
Game内でFrameに変換しているみたいだけど、
このタイミングでFrameにする方が良い気がする。
もっといえばShotは`scores`にある配列の要素それ自体が該当すると思う。
```suggestion
  shots = scores.map { |score| Shot.new(score) }
```